src/VisualStudio/Core/Next/Remote/RemoteHostClientFactory.cs

```diff
@@ -15,20 +14,20 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
     [ExportWorkspaceService(typeof(IRemoteHostClientFactory)), Shared]
     internal class RemoteHostClientFactory : IRemoteHostClientFactory
     {
-        public Task<RemoteHostClient> CreateAsync(Workspace workspace, CancellationToken cancellationToken)
+        public async Task<RemoteHostClient> CreateAsync(Workspace workspace, CancellationToken cancellationToken)
         {
             try
             {
                 // this is the point where we can create different kind of remote host client in future (cloud or etc)
-                return ServiceHubRemoteHostClient.CreateAsync(workspace, cancellationToken);
+                return await ServiceHubRemoteHostClient.CreateAsync(workspace, cancellationToken).ConfigureAwait(false);
```

Comment:
these try/catch should be removed once Jenkins move to preview 5.

and yes, it is supposed to catch the exception before any await which is creating service hub process since this is here so that we can run vsi in jenkins which doesn't support service hub.